src/test/java/net/sf/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -1205,16 +1204,15 @@ public void testCommandLineSinglePdf() throws IOException, TransformerException,
      * Test whether the command-line client can pick one of several entries from a bibtex file
      * @throws IOException
      * @throws TransformerException
-     *
      */
     @Test
-    @Ignore
     public void testCommandLineByKey() throws IOException, TransformerException {
 
         File tempBib = File.createTempFile("JabRef", ".bib");
         try (BufferedWriter fileWriter = Files.newBufferedWriter(tempBib.toPath(), StandardCharsets.UTF_8)) {
             fileWriter.write(t1BibtexString());
             fileWriter.write(t2BibtexString());
+            fileWriter.close();
```

Comment:
Why do you need the close statement here? `fileWrite` is opened in a try-with-resources, so it will be closed at the end.